nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-properties-loader/src/test/groovy/org/apache/nifi/properties/NiFiPropertiesLoaderGroovyTest.groovy

```diff
@@ -119,6 +115,7 @@ class NiFiPropertiesLoaderGroovyTest extends GroovyTestCase {
         assert niFiPropertiesLoader.@keyHex == KEY_HEX
     }
 
+    @Ignore // functionality / test should move to StandardSensitivePropertyProvider class
```

Comment:
Implement comment by moving functionality to referenced test.

Reply:
Migrated test from `NiFiPropertiesLoaderGroovyTest.groovy` to `StandardSensitivePropertyProviderIT.groovy`